{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/3017", "id": 1680024913, "node_id": "I_kwDOJKSTjM5kIyVR", "number": 3017, "title": "Stable 0.2.2 Command browse_website returned Error", "user": {"login": "gregko", "id": 1746812, "node_id": "MDQ6VXNlcjE3NDY4MTI=", "avatar_url": "https://avatars.githubusercontent.com/u/1746812?v=4", "gravatar_id": "", "url": "https://api.github.com/users/gregko", "html_url": "https://github.com/gregko", "followers_url": "https://api.github.com/users/gregko/followers", "following_url": "https://api.github.com/users/gregko/following{/other_user}", "gists_url": "https://api.github.com/users/gregko/gists{/gist_id}", "starred_url": "https://api.github.com/users/gregko/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/gregko/subscriptions", "organizations_url": "https://api.github.com/users/gregko/orgs", "repos_url": "https://api.github.com/users/gregko/repos", "events_url": "https://api.github.com/users/gregko/events{/privacy}", "received_events_url": "https://api.github.com/users/gregko/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5404944721, "node_id": "LA_kwDOJKSTjM8AAAABQijpUQ", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/function:%20browse", "name": "function: browse", "color": "EF8E2B", "default": false, "description": ""}, {"id": 5419307242, "node_id": "LA_kwDOJKSTjM8AAAABQwQQ6g", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/needs%20info", "name": "needs info", "color": "74650E", "default": false, "description": ""}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 3, "created_at": "2023-04-23T13:17:01Z", "updated_at": "2023-04-26T02:36:13Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### \u26a0\ufe0f Search for existing issues first \u26a0\ufe0f\n\n- [X] I have searched the existing issues, and there is no existing issue for my problem\n\n### Which Operating System are you using?\n\nWindows\n\n### Which version of Auto-GPT are you using?\n\nStable (branch)\n\n### GPT-3 or GPT-4?\n\nGPT-4\n\n### Steps to reproduce \ud83d\udd79\n\nRunning on Windows 11 [Version 10.0.22621.1631], python 3.11.3. Asked to design a modern website template for my small software business. It tried to browse my website and gave me this error.\n\n### Current behavior \ud83d\ude2f\n\nEvery time it tries to browse website, this error pops up:\r\n\r\n2023-04-23 08:43:36,523 INFO NEXT ACTION:  COMMAND = browse_website  ARGUMENTS = {'url': 'http://hyperionics.com/', 'question': 'What are the main products and services offered by Hyperionics Technology?'}\r\n2023-04-23 08:43:51,271 INFO SYSTEM:  Command browse_website returned: Error: Message: \r\nStacktrace:\r\nBacktrace:\r\n\tGetHandleVerifier [0x0060DCE3+50899]\r\n\t(No symbol) [0x0059E111]\r\n\t(No symbol) [0x004A5588]\r\n\t(No symbol) [0x004D08F9]\r\n\t(No symbol) [0x004D0AFB]\r\n\t(No symbol) [0x004FF902]\r\n\t(No symbol) [0x004EB944]\r\n\t(No symbol) [0x004FE01C]\r\n\t(No symbol) [0x004EB6F6]\r\n\t(No symbol) [0x004C7708]\r\n\t(No symbol) [0x004C886D]\r\n\tGetHandleVerifier [0x00873EAE+2566302]\r\n\tGetHandleVerifier [0x008A92B1+2784417]\r\n\tGetHandleVerifier [0x008A327C+2759788]\r\n\tGetHandleVerifier [0x006A5740+672048]\r\n\t(No symbol) [0x005A8872]\r\n\t(No symbol) [0x005A41C8]\r\n\t(No symbol) [0x005A42AB]\r\n\t(No symbol) [0x005971B7]\r\n\tBaseThreadInitThunk [0x76077D59+25]\r\n\tRtlInitializeExceptionChain [0x777DB74B+107]\r\n\tRtlClearBits [0x777DB6CF+191]\r\n\n\n### Expected behavior \ud83e\udd14\n\nNo error, just continue processing...\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# Paste your prompt here\r\n```\r\n\n\n### Your Logs \ud83d\udcd2\n\n```log\r\n<insert your logs here>\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/3017/timeline", "performed_via_github_app": null, "state_reason": null}